eng/BootStrapMSBuild.targets

```
@@ -190,6 +190,11 @@
     <Copy SourceFiles="@(InstalledSdks)"
           DestinationFiles="@(InstalledSdks -> '$(BootstrapDestination)Sdks\%(RecursiveDir)%(Filename)%(Extension)')" />
 
+    <!-- The .NET SDK has a dependency on DependencyModel, but relies on having it "next to MSBuild" in the final
+         layout. Copy it from there to our little weirdo bootstrap layout next to the SDK tasks. -->
+    <Copy SourceFiles="$(DOTNET_INSTALL_DIR)\sdk\$(DotNetCliVersion)\Microsoft.Extensions.DependencyModel.dll"
+          DestinationFolder="$(BootstrapDestination)Sdks\Microsoft.NET.Sdk\tools\net6.0" />
```

```suggestion
          DestinationFolder="$(BootstrapDestination)Sdks\Microsoft.NET.Sdk\tools\$(TargetFramework.ToLowerInvariant())" />
```

Based on how `BootstrapDestination` is set in https://github.com/dotnet/msbuild/pull/6548/files#diff-942cef16e96be372318d9687ee9eeeb7bc7b6722b373172ae14e5e63433a9406R12

---

This is different. It's `net6.0` because that's what the SDK is built targeting. That may (or may not) match what our output TF is, but it's not the same as our current TF, so I'd rather not imply that it is.

---

Makes sense in its own weird way. Something we should think about: Defining properties that are passed into places that we hardcode TF's like `net6.0`. Obviously there are issues with that idea and it wouldn't pan out long term (context can easily be different with every instance of `\net6.0\`), but one can dream.